securedrop/tests/pages-layout/test_source.py

```diff
@@ -73,6 +86,38 @@ def test_next_submission_flashed_message(self):
         self._source_submits_a_message()
         self._screenshot('source-next_submission_flashed_message.png')
 
+    def test_source_checks_for_reply(self):
+        self._source_visits_source_homepage()
+        self._source_chooses_to_submit_documents()
+        self._source_continues_to_submit_page()
+        self._source_submits_a_file()
+        self._source_logs_out()
+        self._journalist_logs_in()
+        self._journalist_checks_messages()
+        self._journalist_downloads_message()
+        self._journalist_sends_reply_to_source()
+        self._source_visits_source_homepage()
+        self._source_chooses_to_login()
+        self._source_proceeds_to_login()
+        self._screenshot('source-checks_for_reply.png')
+        self._source_deletes_a_journalist_reply()
+        self._screenshot('source-deletes_reply.png')
```

Comment:
verified

Side note: the codename shows & the **Hide** button does not, at the bottom of the page. This is dealt with by another pull request https://github.com/freedomofpress/securedrop/pull/2322 but I'm surprised because I never noticed that. Maybe related to javascript being disaled ?